Analyser/src/Coverage.js

```diff
@@ -2,8 +2,19 @@
 
 "use strict";
 
+import Config from './Config';
+
+const lineNumberRegex = /\/\S*.js:[\d]*:[\d]*:([\d]*):([\d]*)/;
```

Comment:
/\^(\/\S*.js:[\d]*:[\d]*:([\d]*):([\d]*)\)$/ would probably be better if we have to use regex (More specific)